spinner/exporter/exporter.py

```diff
@@ -1,36 +1,47 @@
+import os
+import textwrap
+
 import nbformat
 from nbconvert import HTMLExporter
 from nbconvert.preprocessors import ExecutePreprocessor
 
 
-def run_reporter(notebook_path, output_path, pkl_db_path=None):
+def run_reporter(notebook_path, pkl_db_path=None):
+    # Resolve folder and base name (remove .pkl extension)
+    pkl_db_folder = os.path.dirname(pkl_db_path)
+    base_name = os.path.splitext(os.path.basename(pkl_db_path))[0]
+
+    # Paths for the new .ipynb and .html
+    new_notebook_path = os.path.join(pkl_db_folder, f"{base_name}.ipynb")
+    new_html_path = os.path.join(pkl_db_folder, f"{base_name}.html")
```

Comment:
I prefer to use pathlib instead of os.path, it's the modern way of handling paths in python.

```suggestion
    # Resolve folder and base name (remove .pkl extension)
    pkl_db_path = Path(pkl_db_path)
    pkl_db_folder = pkl_db_path.parent
    base_name = pkl_db_path.stem  # Remove the extension

    # Paths for the new .ipynb and .html
    new_notebook_path = pkl_db_folder / f"{base_name}.ipynb"
    new_html_path = pkl_db_folder / f"{base_name}.html"
```